lighthouse-core/lib/i18n/i18n.js

```diff
@@ -130,10 +130,66 @@ function lookupLocale(locale) {
  * @param {Record<string, string | number>} [values]
  */
 function _preprocessMessageValues(icuMessage, values = {}) {
-  const clonedValues = JSON.parse(JSON.stringify(values));
+  let clonedValues = JSON.parse(JSON.stringify(values));
   const parsed = MessageParser.parse(icuMessage);
+
+  const elements = _collectAllCustomElementsFromICU(parsed.elements);
+
+  clonedValues = _processParsedElements(Array.from(elements.values()), clonedValues);
+
+  return clonedValues;
+}
+
+/**
+ * Function to retrieve all 'argumentElement's from an ICU message. An argumentElement
+ * is an ICU element that has an argument attached like {varName, number, bytes}. This
+ * differs from 'messageElement's which are just arbitrary text in a message.
+ *
+ * Note: This function will recursively inspect plural elements for nested arguementElements.
+ *
+ * @param {Array<import('intl-messageformat-parser').Element>} elementsList
```

Comment:
typically we just use plurals for arrays of things unless the `list` gives extra info about the value. I like @patrickhulce's earlier suggestion of `icuElements` (or `messageElements`...plain old `elements` is confusing in web-related terminology :) for this, personally.